lib/bunny_storage_client.rb

```diff
@@ -49,6 +52,11 @@ def get_file(storage_zone_name: nil, filename: nil, as: :string)
     request['accept'] = '*/*'
     response = make_request(uri, request)
 
+    if response.code != '200'
+      @logger.error("Failed to get file from #{storage_zone_name}/#{filename}: #{response.code} #{response.body}")
+      return nil
+    end
```

Comment:
```suggestion
    raise StandardError, "#{response.code} #{response.body}" unless success_code?(response.code)
```

Keeping consistent with a single logger statement.

Reply:
Hey throwing an error will break the `exist?` and other methods.

Reply:
I'm not sure about your concern. Here's why:

1. The exist? method has its own independent implementation and doesn't rely on get_file at all.
2. From an execution perspective, the end result will be identical - the rescue block in this method already handles any exceptions by soft failing (returning nil). Adding a raise here won't change that behavior.